build_tools/third_party/sdl2/SDL_config_windows.h

```diff
@@ -227,6 +227,9 @@ typedef unsigned int uintptr_t;
 /* Enable filesystem support */
 #define SDL_FILESYSTEM_WINDOWS 1
 
+/* Disable sensor support */
+#define SDL_SENSOR_DISABLED 0
```

Comment:
Hmm, if I read correctly, this is enabling it?

Reply:
lol, true... I'll change that to `1`
`defined(SDL_SENSOR_DISABLED)` is the check that this is meant to trigger

Reply:
LOL. Okay that explains.

Reply:
Here's a section of the Linux config for reference: https://github.com/google/iree/blob/a33973ebfa725014da83b5905f7dd6a8be4ce3b4/build_tools/third_party/sdl2/SDL_config_linux.h#L201-L216